docs/topics/Community_Resources.md

```diff
@@ -37,7 +37,7 @@ Many of these libraries are represented in the [unofficial, community-driven Dis
 | [Serenity](https://github.com/serenity-rs/serenity)          | Rust       |
 | [AckCord](https://github.com/Katrix/AckCord)                 | Scala      |
 | [Sword](https://github.com/Azoy/Sword)                       | Swift      |
-| [Discordeno](https://github.com/Skillz4Killz/Discordeno)     | TypeScript |
+| [discordeno](https://github.com/discordeno/discordeno)       | TypeScript |
```

Comment:
@Skillz4Killz is the name supposed to be lowercase?

Reply:
Yep